### DISCUSSION TITLE
Emby修改元数据后115网盘出现相同的档案(e.g. nfo/jpg)

### DISCUSSION THREAD

**Discussion by LoserForever88** [AlistGo/alist#3440] [Q&A]
目前用alist+raidrive挂载115到本地，如果每次削刮后手动修改元数据（或再次削刮），115网盘里会出现相同名字的档案（例如.nfo/poster.jpg ），本地则沒有問題维持最新版本的，这个无法覆盖旧档案的问题有什麼方法可以解决？

**Comment by Futsiang** [comment]
I also meet the same problem, using alist+tmm for Xunlei.... It seems when local buffer rewriting to netdisk but the file has not been finalized there for the continuous data cannot be added to the no.(1), then it will automatically rename a no.(2) and so on untill all thread finished the writing action, but this cannot be solved from client. I with the author can solve it expecially for the scrapper wrting. maybe the buffer logic need to be re-checked before upload to cloud through webDav... I am not clear... waiting for solution

**Comment by Futsiang** [comment]
Solved, I encounted this because directly created the virtual disk by windows.
Now I use CloudDrive manage the Alist created folder, can be managed as net-virtual-disk as well through webDav. Better than before.
